sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/cache.scala

```diff
@@ -0,0 +1,21 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.catalyst.plans.logical
+
+/** Mark that the CacheManager can skip this instance */
+trait IgnoreCachedData {}
```

Comment:
we only have a single class in this file, we can rename the file to `IgnoreCachedData.scala`

Reply:
we can also improve the comments
```
/**
 * A [[LogicalPlan]] operator that does not use the cached results stored in CacheManager.
 */
trait IgnoreCachedData extends LogicalPlan
```

Reply:
Updated.